src/axom/inlet/tests/inlet_object.cpp

```diff
@@ -154,19 +164,20 @@ struct FromInlet<FooWithArray>
   }
 };
 
-TEST(inlet_object, array_of_struct_containing_array)
+TYPED_TEST(InletObjectTest, array_of_struct_containing_array)
 {
   std::string testString =
-    "foo = { [4] = { arr = { [1] = 3 }; }, "
-    "        [7] = { arr = { [6] = 2 }; } }";
+    "foo = { [0] = { arr = { [0] = 3 }; }, "
```

Comment:
For Lua arrays, should the first index be `1` ?
Or, is the `0` there to support the other readers?

Reply:
The other readers will read in arrays implicitly starting at zero, so I've explicitly specified the indices so the Lua reader will produce a matching result (since Lua places no restrictions on indices)